test/test_case.py

```diff
@@ -427,6 +427,29 @@ def record(call_time):
         self.assertEqual(call_time, expected)
 
 
+class Test_ClockedTestCase_setUp(asynctest.ClockedTestCase):
+    setUp_was_here = False
+    def setUp(self):
+        self.setUp_was_here = True
+
+    @asyncio.coroutine
+    def test_setUp(self):
+        self.assertEqual(self.setUp_was_here, True)
+        self.assertTrue(hasattr(self, '_time'))  # Was parent setUp() called?
+
+
+class Test_ClockedTestCase_async_setUp(asynctest.ClockedTestCase):
+    setUp_was_here = False
+    @asyncio.coroutine
+    def setUp(self):
+        self.setUp_was_here = True
+
+    @asyncio.coroutine
+    def test_setUp(self):
+        self.assertEqual(self.setUp_was_here, True)
+        self.assertTrue(hasattr(self, '_time'))  # Was parent setUp() called?
```

Comment:
Instead of checking if _time exists, I prefer:

    yield from self.advance(1)
    self.assertEqual(1, self.loop.time())

Even better, you can call `self.advance(1)` in setUp(), and `assertEqual()` in the test.